pkg/config/config_test.go

```diff
@@ -540,46 +540,98 @@ func Test_issuerToChallengeClaim(t *testing.T) {
 }
 
 func TestToIssuers(t *testing.T) {
-	config := &FulcioConfig{
-		OIDCIssuers: map[string]OIDCIssuer{
-			"example.com": {
-				IssuerURL: "example.com",
-				ClientID:  "sigstore",
-				Type:      IssuerTypeEmail,
+	tests := []struct {
```

Comment:
I reworked the tests because within each map in OIDCIssuers and MetaIssuers the order of the response isn't guaranteed. To test the username and URI types, I've added single issuer configs.